public/js/splitjs/splitjs.controller.js

```diff
@@ -0,0 +1,36 @@
+'use strict'; 
+var Split = require('split.js'); 
+module.exports = 
+  angular
+  .module('diSplitJs',[])
+  .controller('diSplitJs', function($scope){
+     alert('wtf');
```

Comment:
I would recommend using `console.log` instead of `alert`. It won't halt the execution of the program, and you can see the values when you open up chrome dev tools